modules/API/API.js

```diff
@@ -18,21 +19,11 @@ let commands = {};
 let initialScreenSharingState = false;
 
 /**
- * JitsiMeetExternalAPI id - unique for a webpage.
- */
-const jitsiMeetExternalApiId
-    = getConfigParamsFromUrl().jitsi_meet_external_api_id;
-
-/**
- * Postis instance. Used to communicate with the external application. If
- * undefined, then API is disabled.
- */
-let postis;
-
-/**
- * Object that will execute sendMessage.
+ * The transport instance used for communication with external apps.
+ *
+ * @type {Transport}
  */
-const target = window.opener || window.parent;
+const transport = getJitsiMeetTransport();
```

Comment:
I'm not sure why `transport` is module global. Technically, until `API.init` passes beyond `this._enabled = true`, `transport` is not utilized, right? So if `API` is disabled, it'll still be one of the reasons for constructing "JitsiMeetTransport".